service/remotecontrol/RemoteControlEvents.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Events fired from the remote control module trough the EventEmitter.
+ */
+
+/**
+ * Notifies about remote control active session status changes.
+ */
+export const ACTIVE_CHANGED = 'active-changed';
```

Comment:
What was the reasoning behind having remote control constants be in service instead of inside modules/remotecontrol?